sensing/autoware_pointcloud_preprocessor/src/outlier_filter/radius_search_2d_outlier_filter_node.cpp

```diff
@@ -28,8 +28,8 @@ RadiusSearch2DOutlierFilterComponent::RadiusSearch2DOutlierFilterComponent(
 {
   // set initial parameters
   {
-    min_neighbors_ = static_cast<size_t>(declare_parameter("min_neighbors", 5));
-    search_radius_ = static_cast<double>(declare_parameter("search_radius", 0.2));
+    min_neighbors_ = declare_parameter<int64_t>("min_neighbors");
```

Comment:
The variable is size_t

Reply:
@knzo25 thanks! However declare_parameter<size_t> doesn't work for ros2 parameter.
![image](https://github.com/user-attachments/assets/b53d23ea-454b-4bc3-858a-4f7d1c28ec04)

reference: https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/

Reply:
I said the same in another PR (that is why I asked for all related PRs be listed):
I know how parameters are implemented in rclcpp, but even though you can not use size_t, the variable is -> implicit cast. This needs do be done explicit

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8474/commits/4df9b9b21b8f87c2fe8cdd4e3c2f9d97ac3c407d